AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin.swift

```diff
@@ -155,7 +155,7 @@ final public class AWSDataStorePlugin: DataStoreCategoryPlugin {
         }
```

Comment:
i don't think we need the check for publisher at line 152 if we are no longer using it